src/modules/collection/index.ts

```diff
@@ -1,18 +1,27 @@
+import assert from 'assert';
 import SolFrenAPI from "../../protocols/solfren-nft";
 import marketplaces from "../../protocols/marketplaces";
 import { API as MarketplaceAPI, CollectionStats } from "../../protocols/marketplaces/types";
-import { CollectionResource } from "./types";
-import { Options } from '../../options';
+import { CollectionResource, ItemResource, ItemOwnerResource } from "./types";
+import { Config } from '../../types';
+import { Connection, PublicKey, ParsedAccountData } from '@solana/web3.js';
+import WonkaAPI from '../../protocols/wonka';
+import { NftEdge } from '../../protocols/wonka/types';
 
 export default class Collection {
   private solFrenAPI: SolFrenAPI;
   private marketplaces: Record<string, MarketplaceAPI> = marketplaces;
+  private connection: Connection;
```

Comment:
NIT: how about
```suggestion
  private solConn: Connection;
```

or `solanaConn`

Reply:
👍🏽